BCI.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<title>BCIs and their applications</title>
+
+	<style>
+
+		html {
+			font-family: Arial;
+			font-size: 15px;
+			color: floralwhite;
+		}
+
+		body {
+			background-color:  #333333;
+		}
+
+		h1 {
+			font-size: 3rem;
+			color: steelblue;
+		}
+
+		h2 {
+			font-size: 3rem;
+			color: royalblue;
+		}
+	</style>
```

Comment:
Could we perhaps move this into its own file and include it using a [`<link>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link) tag?

Reply:
Not sure what you mean?

Reply:
See https://www.w3schools.com/css/css_howto.asp